coffee_core/src/coffee.rs

```diff
@@ -223,7 +224,23 @@ impl PluginManager for CoffeeManager {
         for repo in self.repos.values() {
             if let Some(mut plugin) = repo.get_plugin_by_name(plugin) {
                 log::trace!("{:#?}", plugin);
-                let result = plugin.configure(verbose).await;
+
+                let source_folder = plugin.root_path.clone();
+                let destination_folder =
+                    format!("{}/{}/plugins", self.config.root_path, self.config.network);
+                dir::copy(
+                    source_folder,
+                    destination_folder.clone(),
+                    &dir::CopyOptions::new(),
+                )
+                .map_err(|err| error!("{}", err.to_string()))?;
+
+                let result = plugin
+                    .configure(
+                        &format!("{}/{}", &destination_folder, plugin.name()).to_string(),
+                        verbose,
+                    )
+                    .await;
```

Comment:
```suggestion
                    .await?;
```

and then manage only the path, in this way we simplify the code